### PR TITLE
WIP: RSpecの実行速度を早める

### PR DESCRIPTION
**Pull request by ta-chibana**
# 概要
- `it` ブロックをまとめる方針でテストの実行速度を上げる
- 本来は `it` 内で1件でもfailすると他のテストは実行されないが、設定追加してすべて実行するようにした
- `spec/features/monthly_report_spec.rb`, `spec/feature/admin` 配下だけ適用

# 再現・確認手順

```
$ bundle exec rspec -fd --color --profile
```

### before

```
Top 10 slowest examples (15.39 seconds, 22.7% of total time):
  Admin::Group #update_group_assign should text "菅原 響"
    7.18 seconds ./spec/features/admin/group_spec.rb:61
  Admin::HelpText #index should text "Active Admin Action Logs"
    2.13 seconds ./spec/features/admin/action_log_spec.rb:20
  Admin::Group #update_group_assign should include #<User id: 17722, name: "坂本 真央", employee_code: "336702799596", encrypted_email: "9qR4vCTiLIXXcSRLAZ1...nil, created_at: "2017-05-18 17:05:59", updated_at: "2017-05-18 17:05:59", gender: "gender_unknown">
    0.84481 seconds ./spec/features/admin/group_spec.rb:63
  Admin::Group #update_group_assign should not include #<User id: 17724, name: "高田 美緒", employee_code: "354428363399", encrypted_email: "xo2cg3tQJr3Q5cnNrCI...ted_at: nil, created_at: "2017-05-18 17:06:00", updated_at: "2017-05-18 17:06:00", gender: "female">
    0.83478 seconds ./spec/features/admin/group_spec.rb:64
  Admin::Group #update_group_assign should eq "/admin/groups/18295"
    0.82767 seconds ./spec/features/admin/group_spec.rb:62
  Admin::User #delete_monthly_report retired user reports should be blank
    0.75611 seconds ./spec/features/admin/user_spec.rb:120
  Admin::MonthlyReportTag #destroy should not text "dicta12"
    0.74793 seconds ./spec/features/admin/monthly_report_tag_spec.rb:44
  Admin::MonthlyReportComment #destroy should eq "/admin/monthly_report_comments"
    0.71078 seconds ./spec/features/admin/monthly_report_comment_spec.rb:43
  Admin::Tag batch_action ignoredにする should be ignored
    0.68704 seconds ./spec/features/admin/tag_spec.rb:68
  Admin::User #delete_monthly_report retired user reports should be blank
    0.66573 seconds ./spec/features/admin/user_spec.rb:122

Top 10 slowest example groups:
  Admin::Group
    0.58615 seconds average (16.41 seconds / 28 examples) ./spec/features/admin/group_spec.rb:2
  Admin::HelpText
    0.56305 seconds average (6.19 seconds / 11 examples) ./spec/features/admin/action_log_spec.rb:2
  Admin::MonthlyReportTag
    0.40061 seconds average (4.01 seconds / 10 examples) ./spec/features/admin/monthly_report_tag_spec.rb:2
  Admin::MonthlyReportComment
    0.38194 seconds average (4.2 seconds / 11 examples) ./spec/features/admin/monthly_report_comment_spec.rb:2
  Admin::User
    0.35471 seconds average (13.83 seconds / 39 examples) ./spec/features/admin/user_spec.rb:2
  Admin::UserRole
    0.35095 seconds average (3.86 seconds / 11 examples) ./spec/features/admin/user_role_spec.rb:2
  Admin::Tag
    0.33863 seconds average (5.42 seconds / 16 examples) ./spec/features/admin/tag_spec.rb:2
  Admin::MonthlyWorkingProcess
    0.27694 seconds average (1.66 seconds / 6 examples) ./spec/features/admin/monthly_working_process_spec.rb:2
  Admin::MonthlyReport
    0.27201 seconds average (2.18 seconds / 8 examples) ./spec/features/admin/monthly_report_spec.rb:2
  Admin::Inquiry
    0.26823 seconds average (2.95 seconds / 11 examples) ./spec/features/admin/inquiry_spec.rb:2

Finished in 1 minute 7.78 seconds (files took 12.29 seconds to load)
183 examples, 0 failures
```

### after

```
Top 10 slowest examples (14.68 seconds, 51.8% of total time):
  Admin::Group #update_group_assign should assign user to group
    7.77 seconds ./spec/features/admin/group_spec.rb:69
  Admin::HelpText #index should open the index page
    2.29 seconds ./spec/features/admin/action_log_spec.rb:20
  Admin::User #delete_monthly_report retired user reports should delete the monthly report
    0.6623 seconds ./spec/features/admin/user_spec.rb:133
  Admin::Tag batch_action ignoredにする should update tag status to ignored
    0.65907 seconds ./spec/features/admin/tag_spec.rb:77
  Admin::MonthlyReportComment #destroy should destroy the monthly report comment
    0.63681 seconds ./spec/features/admin/monthly_report_comment_spec.rb:47
  Admin::Tag batch_action fixedにする should update tag status to fixed
    0.6013 seconds ./spec/features/admin/tag_spec.rb:68
  Admin::UserRole #destroy should destroy the user role
    0.59306 seconds ./spec/features/admin/user_role_spec.rb:46
  Admin::MonthlyReportTag #destroy should destroy the monthly report tag
    0.58682 seconds ./spec/features/admin/monthly_report_tag_spec.rb:47
  Admin::HelpText #show should open the show page
    0.44439 seconds ./spec/features/admin/action_log_spec.rb:38
  Admin::User #delete_monthly_report not retired user reports should not text "月報を削除する"
    0.43795 seconds ./spec/features/admin/user_spec.rb:148

Top 10 slowest example groups:
  Admin::Group
    1.37 seconds average (9.6 seconds / 7 examples) ./spec/features/admin/group_spec.rb:2
  Admin::HelpText
    1.37 seconds average (2.74 seconds / 2 examples) ./spec/features/admin/action_log_spec.rb:2
  Admin::MonthlyReportComment
    0.39284 seconds average (1.57 seconds / 4 examples) ./spec/features/admin/monthly_report_comment_spec.rb:2
  Admin::Tag
    0.3665 seconds average (2.2 seconds / 6 examples) ./spec/features/admin/tag_spec.rb:2
  Admin::MonthlyReportTag
    0.34622 seconds average (1.38 seconds / 4 examples) ./spec/features/admin/monthly_report_tag_spec.rb:2
  Admin::UserRole
    0.34376 seconds average (1.72 seconds / 5 examples) ./spec/features/admin/user_role_spec.rb:2
  Admin::User
    0.3403 seconds average (3.4 seconds / 10 examples) ./spec/features/admin/user_spec.rb:2
  Admin::Inquiry
    0.30208 seconds average (0.90624 seconds / 3 examples) ./spec/features/admin/inquiry_spec.rb:2
  Admin::MonthlyWorkingProcess
    0.26504 seconds average (0.53008 seconds / 2 examples) ./spec/features/admin/monthly_working_process_spec.rb:2
  Admin::MonthlyReport
    0.26182 seconds average (0.78546 seconds / 3 examples) ./spec/features/admin/monthly_report_spec.rb:2

Finished in 28.37 seconds (files took 12.43 seconds to load)
62 examples, 0 failures
```

# 対応Issue（任意）
なし

# ToDo

- [x] ラベル付け
- [x] Assigneesで自分を選択
- [ ] レビュアーを2人指定する

### CODE REVIEW

File: spec/features/admin/monthly_report_comment_spec.rb [hunk 1]
```diff
@@ -27,9 +29,11 @@
       click_on '月報コメントを更新'
     end
 
-    it { expect(page).to have_content(new_comment) }
-    it { expect(current_path).to eq admin_monthly_report_comment_path(comment) }
-    it { expect(comment.reload.comment).to eq new_comment }
+    it 'should update the monthly report comment' do
+    expect(page).to have_content(new_comment)
```
Comment (freezer530000) [inline spec/features/admin/monthly_report_comment_spec.rb]: __[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: spec/features/admin/monthly_report_comment_spec.rb [hunk 2]
```diff
@@ -27,9 +29,11 @@
...
+    expect(current_path).to eq admin_monthly_report_comment_path(comment)
```
Comment (freezer530000) [inline spec/features/admin/monthly_report_comment_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/monthly_report_comment_spec.rb [hunk 3]
```diff
@@ -27,9 +29,11 @@
...
+    expect(comment.reload.comment).to eq new_comment
```
Comment (freezer530000) [inline spec/features/admin/monthly_report_comment_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/user_role_spec.rb [hunk 4]
```diff
@@ -7,10 +7,12 @@
 
   describe '#index' do
     before { visit admin_user_roles_path }
-    it { expect(page_title).to have_content('ロール') }
-    it { expect(page).to have_content(user.name) }
-    it { expect(page).to have_content('作成する') }
-    it { expect(page).to have_css('.delete_link') }
+    it 'should open the index page' do
+    expect(page_title).to have_content('ロール')
```
Comment (freezer530000) [inline spec/features/admin/user_role_spec.rb]: __[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: spec/features/admin/user_role_spec.rb [hunk 5]
```diff
@@ -7,10 +7,12 @@
...
+    expect(page).to have_content(user.name)
```
Comment (freezer530000) [inline spec/features/admin/user_role_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/user_role_spec.rb [hunk 6]
```diff
@@ -7,10 +7,12 @@
...
+    expect(page).to have_content('作成する')
```
Comment (freezer530000) [inline spec/features/admin/user_role_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/user_role_spec.rb [hunk 7]
```diff
@@ -7,10 +7,12 @@
...
+    expect(page).to have_css('.delete_link')
```
Comment (freezer530000) [inline spec/features/admin/user_role_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/help_text_spec.rb [hunk 8]
```diff
@@ -28,9 +32,11 @@
       click_on 'ヘルプテキストを更新'
     end
 
-    it { expect(page_title).to have_content("##{help_text.id}") }
-    it { expect(current_path).to eq admin_help_text_path(help_text) }
-    it { expect(page).to have_content(new_body) }
-    it { expect(help_text.reload.body).to eq new_body }
+    it 'should update the help text' do
+    expect(page_title).to have_content("##{help_text.id}")
```
Comment (freezer530000) [inline spec/features/admin/help_text_spec.rb]: __[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: spec/features/admin/help_text_spec.rb [hunk 9]
```diff
@@ -28,9 +32,11 @@
...
+    expect(current_path).to eq admin_help_text_path(help_text)
```
Comment (freezer530000) [inline spec/features/admin/help_text_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/help_text_spec.rb [hunk 10]
```diff
@@ -28,9 +32,11 @@
...
+    expect(page).to have_content(new_body)
```
Comment (freezer530000) [inline spec/features/admin/help_text_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

File: spec/features/admin/help_text_spec.rb [hunk 11]
```diff
@@ -28,9 +32,11 @@
...
+    expect(help_text.reload.body).to eq new_body
```
Comment (freezer530000) [inline spec/features/admin/help_text_spec.rb]: __[rubocop]__
* Inconsistent indentation detected.

**Comment by freezer530000** [comment]
Click here if you want to create rubocop's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/42654368/issues/pull_request/482/diff/fe56b1c009c7078a21df2a7301dd4b55e50a7d15?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by t-tonchim** [comment]
LGTM

**Pull request merged**